### PR TITLE
neatvi 09

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/neatvi.rb [hunk 1]
```diff
@@ -2,14 +2,14 @@ class Neatvi < Formula
...
-      revision: "49e4029728927afb1a22864698be10cf445413aa"
+      tag:      "09",
+      revision: "a3b79df332e3c5804ae57c0348549ff35a69f262"
   head "https://repo.or.cz/neatvi.git"
 
   bottle do
     sha256 cellar: :any_skip_relocation, arm64_big_sur: "a49881c62f05798cf8016e0e6dfb3e00a368c86d338e9ddb332f3f0917e12706"
     sha256 cellar: :any_skip_relocation, big_sur:       "369536734eff4dea2a061a45b61060b9b91400599c2b5149cc56ec889b0c14a5"
-    sha256 cellar: :any_skip_relocation, catalina:      "c2c83bfb47a438d99fb7086a756292a4442dfb1d7eebd0314132814201cd944e"
+    sha256 cellar: :any_skip_relocation, catalina:      "c2c83bfb47a438d99fb7096a756292a4442dfb1d7eebd0314132814201cd944e"
```
Comment (iMichka) [inline Formula/neatvi.rb:12]: Why?

**Review by iMichka — DISMISSED** [review]
There seem to be a change to the bottle block, not sure why.

File: Formula/neatvi.rb (hunk 1 above)
Comment (nandahkrishna) [reply Formula/neatvi.rb:12]: I think Rui's script replaced 08 to 09 everywhere, something similar happened in a previous PR.

Comment (chenrui333) [reply Formula/neatvi.rb:12]: exactly, the command replaced the 08 everywhere. it happened before. Correcting it now.

**Comment by chenrui333** [comment]
<details>
<summary>license header ref `vi.c`</summary>

```
/*
 * NEATVI Editor
 *
 * Copyright (C) 2015-2019 Ali Gholami Rudi <ali at rudi dot ir>
 *
 * Permission to use, copy, modify, and/or distribute this software for any
 * purpose with or without fee is hereby granted, provided that the above
 * copyright notice and this permission notice appear in all copies.
 *
 * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
 * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
 * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
 * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
```

</details>

**Review by cho-m — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**